core/src/Cabana_CommunicationPlan.hpp

```diff
@@ -12,7 +12,7 @@
 #ifndef CABANA_COMMUNICATIONPLAN_HPP
 #define CABANA_COMMUNICATIONPLAN_HPP
 
-#include <CabanaCore_config.hpp>
+#include <Cabana_config.hpp>
```

Comment:
Did you mean to include `Cabana_Utils.hpp`

Reply:
Fixed